src/App.tsx

```diff
@@ -1,7 +1,28 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+export class App extends React.Component {
+  state = {
+    pressedKey: 'Nothing was pressed yet',
```

Comment:
`pressedKey` looks like page message, not actually a pressed key. It's would be better to store in the state `pressedKey` value (as default you can store `' ' `or `null` ) and render the page message according to this state

Reply:
thanks, fixed